velox/common/memory/Memory.cpp

```diff
@@ -60,13 +68,51 @@ uint16_t MemoryManager::alignment() const {
   return alignment_;
 }
 
-MemoryPool& MemoryManager::getRoot() const {
-  return *root_;
+std::shared_ptr<MemoryPool> MemoryManager::getPool(
+    const std::string& name,
+    MemoryPool::Kind kind,
+    int64_t cap) {
```

Comment:
nit: perhaps, use a more descriptive name for this parameter; maybe, maxBytes

Reply:
Changed.